packages/palette/src/elements/LabeledInput/LabeledInput.tsx

```diff
@@ -24,7 +25,7 @@ export const LabeledInput: React.ForwardRefExoticComponent<
 
   const variant: TextVariant = useThemeConfig({ v2: "small", v3: "xs" })
 
-  const isText = typeof label === "string" || typeof label === "number"
+  const isText = isSimpleChildren(label)
```

Comment:
I guess `isText` might be a better name for this helper?

Reply:
Yah i was gonna say the same becuse that's basically all we're checking 👍

Reply:
:renamed: